content/en/docs/concepts/scheduling-eviction/resource-bin-packing.md

```diff
@@ -3,58 +3,82 @@ reviewers:
+## Enabling Bin Packing using MostAllocated
+`MostAllocated` allows the users to specify the resources with weight for each resources to 
+score nodes on the request to capacity ratio. It favors nodes that have a high allocation of resources.
 
-Kubernetes allows the users to specify the resources along with weights for
+```yaml
+apiVersion: kubescheduler.config.k8s.io/v1beta3
+kind: KubeSchedulerConfiguration
+profiles:
+- pluginConfig:
+  - args:
+      scoringStrategy:
+        resources:
+        - name: intel.com/foo
+          weight: 3
+        - name: intel.com/bar
+          weight: 3
+        type: MostAllocated
+    name: NodeResourcesFit
+```
+
+**This feature is disabled by default**
+
+## Enabling Bin Packing using RequestedToCapacityRatio
+
+`RequestedToCapacityRatio` allows the users to specify the resources along with weights for
 each resource to score nodes based on the request to capacity ratio. This
 allows users to bin pack extended resources by using appropriate parameters
-and improves the utilization of scarce resources in large clusters. The
-behavior of the `RequestedToCapacityRatioResourceAllocation` priority function
-can be controlled by a configuration option called `RequestedToCapacityRatioArgs`. 
-This argument consists of two parameters `shape` and `resources`. The `shape` 
+and improves the utilization of scarce resources in large clusters. It favors nodes according to a
+configured function of the allocated resources. The
+behavior of the `RequestedToCapacityRatio` in `NodeResourcesFit` score function
+can be controlled by a configuration option called `scoringStrategy`. 
+This argument consists of two parameters `requestedToCapacityRatioParam` and 
+`resources`. The `shape` in `requestedToCapacityRatioParam` 
 parameter allows the user to tune the function as least requested or most 
 requested based on `utilization` and `score` values.  The `resources` parameter 
 consists of `name` of the resource to be considered during scoring and `weight` 
 specify the weight of each resource.
 
 Below is an example configuration that sets
-`requestedToCapacityRatioArguments` to bin packing behavior for extended
+`RequestedToCapacityRatio` to bin packing behavior for extended
 resources `intel.com/foo` and `intel.com/bar`.
 
 ```yaml
 apiVersion: kubescheduler.config.k8s.io/v1beta3
 kind: KubeSchedulerConfiguration
 profiles:
-# ...
-  pluginConfig:
-  - name: RequestedToCapacityRatio
-    args: 
-      shape:
-      - utilization: 0
-        score: 10
-      - utilization: 100
-        score: 0
-      resources:
-      - name: intel.com/foo
-        weight: 3
-      - name: intel.com/bar
-        weight: 5
+- pluginConfig:
+  - args:
+      scoringStrategy:
+        resources:
+        - name: intel.com/foo
+          weight: 3
+        - name: intel.com/bar
+          weight: 3
+        requestedToCapacityRatioParam:
+          shape:
+          - utilization: 0
+            score: 10
+          - utilization: 100
```

Comment:
I'm a bit confused here. Shouldn't higher utilization have the higher score? Or is lower score better?

Reply:
Yes. You are right.  There is an explanation below. But I think it's better to change it.

![image](https://user-images.githubusercontent.com/8958057/168780801-a6eeb495-cf35-49f8-9b73-fcfe19f63bfe.png)
https://kubernetes.io/docs/concepts/scheduling-eviction/resource-bin-packing/